AK/Time.h

```diff
@@ -456,6 +461,67 @@ class UnixDateTime : public Detail::UnawareTime {
     }
 };
 
+// Monotonic time represents time returned from the CLOCK_MONOTONIC clock, which has an arbitrary fixed reference point.
+class MonotonicTime : private Detail::UnawareTime {
```

Comment:
What makes this conceptually an UnawareTime and not a Duration?
Not asking for a change, just trying to understand.

Reply:
UnawareTime provides some internal APIs

Reply:
Also: Duration represents a time span, not a point in time. MonotonicTime is a point in time, even though that point in time is represented by a Duration since the kernel-determined starting point. The point in time base class here is UnawareTime.